manifests/kustomize/env/aws/OWNERS

```diff
@@ -1,3 +1,4 @@
 approvers:
   - Jeffwan
   - PatrickXYS
+  - akartsky
```

Comment:
Hello @akartsky , you are adding yourself to the AWS owner file. Do you mind having a separate PR only for this change and ask for Jeffwan or PatrickXYS approval on this? Thank you!